lightning/tests/lightning_checkpoint_chunks/run.sh

```diff
@@ -116,7 +116,7 @@ check_contains "sum(i): $(( $ROW_COUNT*$CHUNK_COUNT*(($CHUNK_COUNT+2)*$ROW_COUNT
 [ -e "$TEST_DIR/cpch.pb.1234567890.bak" ]
 
 ## default auto analyze tick is 3s
```

Comment:
@Leavrth Do we have to check this value? Can we skip the waiting for the auto-analyze and check if the modified count is 5000?

Reply:
cc @lance6716

Reply:
the purpose is to make sure lightning physical import will trigger auto analyze in the future. We can have other way to check it. sleep 130s is unacceptable.

Reply:
> I think it's OK if stats team has enough confident that "modified count is 5000" will trigger an auto analyze eventually.

I think we still need to verify this behavior. I will try to use a failpoint to trigger the auto-analyze check immediately.